### ISSUE TITLE
Core Developer: Quick links component

### ISSUE THREAD

**Issue by ineedsubstance** [punchcard-cms/punchcard#541]

**Comment by mkiang3051** [comment]
Nothing to add to this story - I modeled my `Quick add` after this concept of allowing users to quickly and easily access content types they have used most often. It's just a sticky bar/menu at the bottom of the mobile screen. It was a first pass idea to try to have something easily accessible to add something quickly rather than having to scroll down to a section that a user may not even see on their mobile/desktop screen. 

I was just modifying the way it shows up on a mobile device to make it more mobile friendly - since you wouldn't be able to see this section on mobile if the user has several items in the two tables above this section. And also, on desktop, it may require the user to scroll. So I thought having it be sticky or maybe surfaced where someone can always see it would be helpful.

I'll paste a copy of my mock here as well for reference.

![mobile dash v1 mk](https://cloud.githubusercontent.com/assets/5656022/21283621/22dc0d54-c3cb-11e6-8c10-9e81dd90e148.png)

**Comment by ineedsubstance** [comment]
@mkiang3051 - I like this idea and think it could work on all size screens. 

1. What happens when you click on the `+` button?
2. Let's think about the number of steps and how to minimize them as much as possible. Essentially, is this easier than the user clicking on content and then selecting the content type? 
3. What does this look like on desktop assuming we take a similar approach?

**Comment by mkiang3051** [comment]
Yep! I can do some more work/thinking on this after secondary nav. 👍
